src/CMakeLists.txt

```diff
@@ -144,8 +144,16 @@ if("${CMAKE_SOURCE_DIR}/src" STREQUAL PROJECT_SOURCE_DIR)
   message(STATUS "Checkpoint: building as top-level")
 else()
   message(STATUS "Checkpoint: building as a sub-directory")
-  install(TARGETS detector EXPORT ${CHECKPOINT_LIBRARY})
-  set(DETECTOR_EXPORT detector)
+
+  if(CMAKE_VERSION VERSION_LESS "3.13.0")
+    message(
+      WARNING "Checkpoint: Install targets skipped, effects packages built with a Checkpoint install"
```

Comment:
This warning should probably include a statement of the CMake version dependence and an issue reference in it

Reply:
Fixed